contrib/inventory/puppetdb.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+#
```

Comment:
Sorry I missed this earlier. Please include a copyright and license notice, example here:

https://github.com/ansible/ansible/blob/devel/contrib/inventory/azure_rm.py#L3